frontend/i18n/en-US.json

```diff
@@ -370,6 +370,19 @@
   "pages.events.search.header-title": "Events search",
   "pages.events.search.subheader": "Find events to attend",
   "pages.events.team.tagline": "Event organizers",
+  "pages.resources.create.header": "Information",
+  "pages.resources.create.subtext": "Thanks for linking a resource to activist! This is how you can keep all the tools and information that you need to get active in one place. Please only link resources that you have the right to share with others. You can learn more about our terms and confitions for copyrighted materials on the resources page of the documentation",
+  "pages.resources.create.resource-name": "Name",
+  "pages.resources.create.resource-name-placeholder": "A name for the resource",
+  "pages.resources.create.link": "Link",
+  "pages.resources.create.link-placeholder": "A URL to the resource",
+  "pages.resources.create.description": "Short description",
+  "pages.resources.create.description-placeholder": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua",
+  "pages.resources.create.location":"Location",
```

Comment:
Note that this should be combined with the other "Location" i18n key. Ping @to-sta that an error for the workflow wasn't triggered :)

Reply:
Should i use the organization location on this key, or add Location to global keys?

Reply:
You can check the Python script that was supposed to fail on the branch and should fail locally :) Run `python frontend/i18n/check_repeat_i18n_values.py` and it should list this values that have been repeated and also suggest what the lowest level of key should be.

Reply:
Looks like we can do `"pages._global.location": "Location"` :) Give it a check though 😊

Reply:
Nice @IsaacDMz :) Thanks! I'll get to this in the coming days. My home internet's a bit screwed up atm, and I don't like reviewing on my phone's hotspot 😅